frontend/src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap/dist/css/bootstrap-theme.css';
+import './override.less';
+import './index.css';
```

Comment:
could we write only `.less` code, as in `cat`?

Reply:
I would skip this and merge as it is. All the machinery to write `less` is in place, this file can be changed at any moment when we start to write styles.

Reply:
Done in 3d57d81.